test/math/QuaternionTest.ooc

```diff
@@ -138,7 +161,9 @@ QuaternionTest: class extends Fixture {
 			pitch := Float toRadians(-30.0f)
 			yaw := Float toRadians(45.0f)
 			quaternion := Quaternion createRotationZ(yaw) * Quaternion createRotationY(pitch) * Quaternion createRotationX(roll)
-			expLog := quaternion exponential logarithm
+			//expLog := quaternion exponential logarithm
+			exp := (quaternion exponential)
+			expLog := exp logarithm
```

Comment:
This makes me sad. Does `expLog := (quaternion exponential) logarithm` also not work?